docs/core/deploying/index.md

```diff
@@ -1,98 +1,173 @@
 
-It also has a number of disadvantages:
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+<br/>
 
-- The size of your deployment package is relatively large, since you have to include .NET Core as well as your app and its third-party dependencies.
+### Step 2: Publish your app
 
-  Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md). Ordinarily, .NET Core on Linux relies on the [ICU libraries](http://icu-project.org) for globalization support. In invariant mode, the libraries are not included with your deployment, and all cultures behave like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
+You can publish your app in one of two configurations - as a [Framework Dependent Deployment (FDD)](publish.md#Framework-Dependent-Deployment-FDD) or as a [Self-Contained Deployment (SCD)](publish.md#Self-contained-deployment-SCD).
 
-- Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+For both of the above configurations, you can publish the app either as a set of loose assemblies for either `xcopy` or package up for deployment. Alternatively, you can publish your app as a single file executable.
 
-## Framework-dependent executables (FDE)
+More information about each of these configurations can be found in the document on how to [Publish your app](publish.md).
 
-Starting with .NET Core 2.2, you can deploy your app as an FDE, along with any required third-party dependencies. Your app will use the version of .NET Core that's installed on the target system.
 
-### Why deploy a framework-dependent executable?
+<br/>
 
-Deploying an FDE has a number of advantages:
+### Step 3: Package your app for deployment
 
-- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+Packaging your app is optional. You can `xcopy` the publish folder of your app to a folder on the target computer. 
 
-- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
+If you prefer to package your app with a more professional installer experience instead of an `xcopy` based deployment, you have a couple of options.
 
-- Your app can be run by calling the published executable without invoking the `dotnet` utility directly.
+You can package your app using one of the deployment technologies built into Visual Studio (MSIX and MSI), or use a third-party installer of your choice.
```

Comment:
You are really only talking about desktop apps here.  You can package other apps in numerous ways, linus packages, docker images, snaps, etc.  This section should be more inclusive.